geode-core/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/ExportClusterConfigurationCommandDUnitTest.java

```diff
@@ -86,4 +94,60 @@ public void getClusterConfigWithFile() throws IOException {
     assertThat(content).startsWith("<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\"?>")
         .contains("<region name=\"regionA\">");
   }
+
+  @Test
+  public void testExportWithAbsolutePath() throws Exception {
+    Path exportedZipPath =
+        tempFolder.getRoot().toPath().resolve("exportedCC.zip").toAbsolutePath();
+
+    testExportClusterConfig(exportedZipPath.toString());
+  }
+
+  @Test
+  public void testExportWithRelativePath() throws Exception {
+    testExportClusterConfig("mytemp/exportedCC.zip");
+    FileUtils.deleteQuietly(new File("mytemp"));
```

Comment:
For this and the following test `mytemp` is created in the `tempFolder` so explicit deletion shouldn't be necessary.

Reply:
actually the exported files without absolute paths is written to the user.dir, so we need an explicit delete both here and the test below. The test below is deleting the wrong file. I checked a fix for that.

Reply:
Thanks. I stand corrected.